### DISCUSSION TITLE
How to do an "notice" box

### DISCUSSION THREAD

**Discussion by jehoshua7** [razonyang/hugo-theme-bootstrap#211] [Q&A] [answered]
I have been testing the [hugo-theme-relearn](https://github.com/McShelby/hugo-theme-relearn) and it does an info box as can be seen at https://relearn.netlify.app/

The **md** code was 

`{{% notice info %}} The theme initially was a fork of the great .......{{% /notice %}}`

The generated **htm**l is

> <div class="notices info">
    <div class="label">Info</div><p>The theme initially was a fork of the great <a href="https://github.com/matcornic/hugo-theme-learn">Learn theme</a> with the aim of fixing long outstanding bugs and adepting to latest Hugo features. As far as possible this theme tries to be a drop-in replacement for Learn theme.</p>
</div>

Is there a method to do this in the bootstrap theme ? Possibly something like a one row/one column **html** table ?

**Comment by razonyang** [comment, marked as answer]
It can be implemented by creating [custom shortcode](https://gohugo.io/templates/shortcode-templates/).

```html
// layouts/shortcodes/notice.html
<div class="label">Info</div><p>{{ .Inner }}</p>
```

```markdown
// post.md
{{< notice >}}
msg
{{< /notice >}}
```

**Comment by jehoshua7** [comment]
@razonyang - Thanks, the code works fine.
